### PR TITLE
Follow-up for #4165

### PR DESCRIPTION
**Pull request by glitsj16**
Another batch of comments rewording to encourage using .local overrides.

### CODE REVIEW

File: etc/profile-m-z/nano.profile [hunk 1]
```diff
@@ -47,7 +47,7 @@ x11 none
 private-bin nano,rnano
 private-cache
 private-dev
-# Comment the next line if you want to edit files in /etc directly
+# Add 'ignore private-etc' to your nano.local if you want to edit files in /etc directly.
 private-etc alternatives,nanorc
```
Comment (rusty-snake) [inline etc/profile-m-z/nano.profile:51]: OT: Does this work w/o `writeable-etc`?

File: etc/profile-m-z/ostrichriders.profile [hunk 2]
```diff
@@ -43,7 +43,7 @@ tracelog
 disable-mnt
 private-bin ostrichriders
 private-cache
-# comment the following line if you need controller support
+# Add 'ignore private-dev' to your ostrichriders.local if you need controller support.
```
Comment (rusty-snake) [inline etc/profile-m-z/ostrichriders.profile:46]:
```suggestion
# Add 'ignore noinput' to your ostrichriders.local if you need controller support.
```

File: etc/profile-m-z/steam.profile [hunk 3]
```diff
@@ -131,34 +131,36 @@ caps.drop all
 #ipc-namespace
 netfilter
 nodvd
-# nVidia users may need to comment / ignore nogroups and noroot
+# If you use nVidia you might need to add 'ignore nogroups' and 'ignore noroot' to your steam.local.
```
Comment (rusty-snake) [inline etc/profile-m-z/steam.profile:134]:
```suggestion
# If you use nVidia you might need to add 'ignore noroot' to your steam.local.
```

623e6821 + cb460c32

File: etc/profile-m-z/steam.profile [hunk 4]
```diff
@@ -131,34 +131,36 @@ caps.drop all
...
 private-dev
-# private-etc breaks a small selection of games on some systems, comment to support those
+# private-etc breaks a small selection of games on some systems. Add 'ignore private-etc'
+# to your steam.local to support those.
 private-etc alsa,alternatives,asound.conf,bumblebee,ca-certificates,crypto-policies,dbus-1,drirc,fonts,group,gtk-2.0,gtk-3.0,host.conf,hostname,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,localtime,lsb-release,machine-id,mime.types,nvidia,os-release,passwd,pki,pulse,resolv.conf,services,ssl
 private-tmp
 
-# breaks appindicator support
-# dbus-user none
+# dbus-user none - breaks appindicator support
```
Comment (rusty-snake) [inline etc/profile-m-z/steam.profile:165]: And a lot more. Maybe just `# dbus-user none`.

**Review by rusty-snake — APPROVED** [review]

File: etc/profile-m-z/steam.profile (hunk 3 above)
Comment (glitsj16) [reply etc/profile-m-z/steam.profile]: Fixed, thanks.

File: etc/profile-m-z/steam.profile (hunk 4 above)
Comment (glitsj16) [reply etc/profile-m-z/steam.profile]: Fixed. Thanks.

File: etc/profile-m-z/ostrichriders.profile (hunk 2 above)
Comment (glitsj16) [reply etc/profile-m-z/ostrichriders.profile]: Fixed, thanks.

File: etc/profile-m-z/nano.profile (hunk 1 above)
Comment (glitsj16) [reply etc/profile-m-z/nano.profile:53]: Nice catch, forgot I have that in my nano.local. For good measure I added a similar comment for editing files in /var too. What do you think?

File: etc/profile-m-z/nano.profile [hunk 5]
```diff
@@ -47,10 +47,14 @@ x11 none
...
+#ignore private-etc
+#writable-etc
 private-etc alternatives,nanorc
 
 dbus-user none
 dbus-system none
 
 memory-deny-write-execute
+# Add the next line to your nano.local if you want to edit files in /var directly.
+#writable-var
```
Comment (rusty-snake) [inline etc/profile-m-z/nano.profile:60]: https://github.com/netblue30/firejail/blob/b0c3dea0f676d46f2843ddb47a61ce68f3d6cb01/etc/templates/profile.template#L195-L199

File: etc/profile-m-z/nano.profile (hunk 1 above)
Comment (glitsj16) [reply etc/profile-m-z/nano.profile:53]: Moved the writable-var comment cfr. profile.template. Thanks for your thorough review!

**Pull request merged**